iOS Client/PodcastsController/PodcastFilterViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  PodcastFilterViewController.swift
+//  iOS Client
+//
+//  Created by Jonathan Witten on 4/9/18.
+//  Copyright © 2018 Mockingbird. All rights reserved.
+//
+
+import Foundation
+import ReSwift
+import UIKit
+
+class PodcastsFilterViewController: UIViewController, UITableViewDataSource, UITableViewDelegate, StoreSubscriber {
+    
+    @IBOutlet weak var tableView: UITableView!
+    var visibleStreams: Set<PodcastStream> = Set<PodcastStream>()
+    var streams: [PodcastStream] = [.pz, .mockingCast, .mockingPulpit]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.register(UINib(nibName: PodcastFilterTableViewCell.reuseIdentifier, bundle: nil), forCellReuseIdentifier: PodcastFilterTableViewCell.reuseIdentifier)
```

Comment:
My concern with making the reuse identifier match the nib name is the two sources of truth. We're now in a situation where if either the reuse identifier changes or the name of the view file changes then this code will break.

Reply:
if we hard-code the nib name, then at least changing the reuse identifier doesn't break the code.